ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/EthGetTransactionByHashTest.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright ConsenSys AG.
```

Comment:
nit: new files should be Copyright contributors to Hyperledger Besu

Reply:
done